app/scripts/directives/realtimestring.js

```diff
@@ -53,7 +53,7 @@ angular.module('oncokbApp')
                         });
                     }
                     scope.$watch('data[key]', function (n, o) {
-                        if (!_.isUndefined(n)) {
+                        if (!_.isUndefined(n) && n !== null) {
```

Comment:
I received an error message from sentry and it showed that n is null. I tried to regenerate this issue but I failed. In case, it happens again I added a check on `n`.